### ISSUE TITLE
ffmpeg `GLIBC_2.27' not found error

### ISSUE THREAD

**Issue by carlosramireziii** [heroku/heroku-buildpack-activestorage-preview#13]
Starting a few days ago, the following error started coming up on our Heroku apps which are using this build pack.

```
ffmpeg: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.25' not found (required by ffmpeg)
ffmpeg: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.27' not found (required by ffmpeg)
ffmpeg: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.27' not found (required by ffmpeg)
```

**Comment by cji** [comment]
@carlosramireziii working on a fix for ffmpeg here https://github.com/heroku/sffmpeg/pull/8 and then will update the buildpack

**Comment by gsaada** [comment]
@cji Thanks for your response.
So I can do nothing to solve it till @carlosramireziii will update the buildpack?

**Comment by cji** [comment]
@gsaada upgrading to the heroku-18 stack should also fix this issue for you with the current buildpack. https://devcenter.heroku.com/articles/upgrading-to-the-latest-stack

**Comment by gsaada** [comment]
@cji Thank you for your response.
I prefer not to upgrade, I'll need to test everything to make sure nothing break.
Do you have an ETA for the fix to be approved?

**Comment by cji** [comment]
The buildpack change has been merged in #14 and a new deploy should include the fix in your heroku-16 based applications. 

I added in some cache busting to address issues we saw with this in #9, but if you still don't see the new version of ffmpeg in your app after a deploy, try walking through https://help.heroku.com/18PI5RSY/how-do-i-clear-the-build-cache
